recipes/intake-sqlite/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "intake-sqlite" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/intake-sqlite-{{ version }}.tar.gz
+  sha256: f70f357f0686753ae58e5a0d95007adcb4e2c283a80a13852d103c83470925f7
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<3.11
+    - setuptools-scm
+  run:
+    - fsspec <2022.6.0
+    - intake <=0.6.5
+    - intake-sql <=0.3.1
+    - pandas <=1.4.2
+    - python >=3.8,<3.11
+    - sqlalchemy <=1.4.36
+
+test:
+  imports:
+    - intake_sqlite
+    - package_data
```

Comment:
While this is in the upstream package it can be confusing when a single package installs two modules, specially when one name is generic and unrelated to the main package. This can get easily clobbered if other packages start putting their data in the "root" of site-packages as a separated module. (We already have that situation for "tests" module.)

While we can merge I strongly recommend you opening an issue upstream and asking for better practices for shipping data. (Either inside the main module, as a submodule, or in a share or config folder.)

Reply:
Whoops, that's definitely a misplaced directory in the template repo I used to create the `intake-sqlite` repo. I will fix it in the template and `intake-sqlite` and do another release. Thanks for pointing it out.